modules/renter/contractor/host_integration_test.go

```diff
@@ -999,6 +1001,7 @@ func TestIntegrationCachedRenew(t *testing.T) {
 
 	// corrupt the contract and cachedRevision
 	badContract := contract
+	badContract.GoodForRenew = true
```

Comment:
Needed to add these to avoid triggering the sanity checks I added to make sure we only renew GoodForRenew contracts.